node/src/main/scala/org/bitcoins/node/config/NodeAppConfig.scala

```diff
@@ -143,6 +143,25 @@ case class NodeAppConfig(baseDatadir: Path, configOverrides: Vector[Config])(
     else 1
   }
 
+  lazy val enablePeerDiscovery: Boolean = {
+    if (config.hasPath("bitcoin-s.node.enable-peer-discovery"))
+      config.getBoolean("bitcoin-s.node.enable-peer-discovery")
+    else false
+  }
+
+  // https://github.com/lightbend/config/blob/master/HOCON.md#duration-format
+  lazy val peerDiscoveryTimeout: Duration = {
+    if (config.hasPath("bitcoin-s.node.peer-discovery-timeout"))
+      config.getDuration("bitcoin-s.node.peer-discovery-timeout")
+    else Duration.ofMinutes(10)
+  }
+
+  lazy val useDefaultPeers: Boolean = {
+    if (config.hasPath("bitcoin-s.node.use-default-peers"))
+      config.getBoolean("bitcoin-s.node.use-default-peers")
+    else true
```

Comment:
so by default if we merge this PR, we will obtain the original behavior of the node because this is `true`, correct? 

When we want to "turn on" this PR by default, we will adjust this setting to `false`?

Reply:
FWIW, i think we can remove #4404 when this PR gets merged.

The problem i had on #4404 was in docker environments and resolving the configuration relative to our `bitcoin-s.network` inside of `docker-application.conf`

There was no way to configure a hard coded peer based on network in `docker-application.conf` and I needed a way to get a user unblocked, thus #4404

Reply:
It would be helpful to have some documentation on the configuration options you added and what they do. Can you add that to the example configuration file we have in `configuration.md` ? 

https://bitcoin-s.org/docs/config/configuration#example-configuration-file

Reply:
Added this and some other configs.